inc/class-wpseo-options.php

```diff
@@ -2791,192 +2791,195 @@ protected function clean_option( $option_value, $current_version = null, $all_ol
 /*******************************************************************
  * Option: wpseo_ms
  *******************************************************************/
-if ( is_multisite() ) {
```

Comment:
If this conditional is removed, the `WPSEO_Options` constructor will need to be adjusted as it now checks for whether the class exists before creating/loading that option which would no longer be a valid test once you remove this.